### PR TITLE
ci: require unit tests to pass with node 12, 14, 15

### PR DESCRIPTION
**Pull request by tylerlevine**
Otherwise, we might see some regressions which break these
newly-supported versions.

Ticket: BG-23173

### CODE REVIEW

File: .drone.yml
Comment (ericcrosson-bitgo) [inline .drone.yml:350]: What's this signature?

File: .drone.jsonnet
Comment (ericcrosson-bitgo) [inline .drone.jsonnet:233]: oooooooooo very cool

**Review by ericcrosson-bitgo — APPROVED** [review]

File: .drone.yml
Comment (tylerlevine) [reply .drone.yml:350]: It's this handy dandy feature: https://readme.drone.io/signature/

> You can optionally sign your configuration file to verify authenticity and prevent tampering. This is useful if your repository is public and you need to prevent unauthorized changes to your configuration.
\
If a user modifies the configuration and signature verification fails, the pipeline is blocked pending manual approval by an authorized user with write or administrative access to the repository.

This is very useful for us so that users who don't have write access can't modify the build scripts to dump secrets :lock:

Comment (ericcrosson-bitgo) [reply .drone.yml:350]: whoa sweet! Thanks for the link :slightly_smiling_face:

**Pull request merged**
